### PR TITLE
Add Spanish translation for services page

### PR DESCRIPTION
**Pull request by CarlosMChica**

### CODE REVIEW

File: _i18n/es.yml [hunk 1]
```diff
@@ -140,6 +140,47 @@ publications-tweet_feed:
 publications-post:
    author_title: Por
    date_title: Publicado
+services-software_creation:
+   title: Creación de software
+   subtitle: <p><strong>Te ayudaremos a crear software bien elaborado que aporta valor a su negocio.</strong></p>
```
Comment (alvarogarcia7) [inline _i18n/es.yml]: Difference in style: tú vs usted. please pick one and keep it consistent

File: _i18n/es.yml [hunk 2]
```diff
@@ -140,6 +140,47 @@ publications-tweet_feed:
...
+           <p>Nuestros craftsmen han perfeccionado su destreza en tecnologías y prácticas para poder entregar a tiempo y con calidad. Nos ajustamos a sus necesidades.</p>
```
Comment (alvarogarcia7) [inline _i18n/es.yml:147]: Not really sure if 'craftsmen' is understood by Spanish speakers

File: _i18n/es.yml [hunk 3]
```diff
@@ -140,6 +140,47 @@ publications-tweet_feed:
...
+services-software_creation-embedded_craftsmen:
+   title: Craftsmen Embebidos
```
Comment (alvarogarcia7) [inline _i18n/es.yml:149]: same here

File: _i18n/es.yml (hunk 1 above)
Comment (CarlosMChica) [reply _i18n/es.yml]: @alvarogarcia7 I tried to keep consistency and use 'usted'. However, I may have missed some, so please let me know if you spot some other occurrences.

File: _i18n/es.yml (hunk 2 above)
Comment (CarlosMChica) [reply _i18n/es.yml:147]: @alvarogarcia7 I also wondered but I think it will be better understood than using something like 'artesano'

**Comment by CarlosMChica** [comment]
services codurance

**Pull request merged**